ogc-rs/src/lib.rs

```diff
@@ -66,11 +69,22 @@ pub mod prelude {
 
     // Export Services
     pub use crate::console::*;
+    pub use crate::constants::*;
     pub use crate::debug::*;
+    pub use crate::gu::*;
+    pub use crate::gx::*;
+    pub use crate::pad::*;
     pub use crate::system::*;
+    pub use crate::tpl::*;
     pub use crate::video::*;
+    pub use crate::wpad::*;
     pub use crate::{print, println};
 
+    // Types
+    pub type Mtx = [[f32; 4]; 3];
+    pub type Mtx34 = [[f32; 4]; 3];
+    pub type Mtx44 = [[f32; 4]; 4];
```

Comment:
These types could be moved into the ``gx.rs`` file and then referenced here for neatness.

Reply:
Moved to `gx.rs` as `pub`.